definitions/infra-container/definition.yml

```diff
@@ -3,25 +3,38 @@ type: CONTAINER
 
 synthesis:
   rules:
+  # Dockerstats receiver from otel collector.
   - identifier: container.id
     name: container.name
     encodeIdentifierInGUID: true
-    # This is a temporary condition to restrict synthesis for a particular set of telemetry data.
     conditions:
-      - attribute: nr.entity_type
-        value: container
```

Comment:
we have removed this condition that required to add this extra attribute in the metrics, and it was used for development. 
The impact of this has been analyzed [here](https://newrelic.atlassian.net/wiki/spaces/INST/pages/2865759989/OTEL+docker%3A+entities+and+golden+metrics+experience?focusedCommentId=2874117934)